ansible/roles/executor/templates/stop.sh.j2

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-ps -e | grep executor-{{ cluster_name }} | awk '{ print $1 }' | xargs -I _ kill -9
+ps -e | grep executor-{{ cluster_name }} | awk '{ print $1 }' | xargs -r kill -9
```

Comment:
Can we use `kill` instead of `kill -9` here?

Reply:
addressed